documentation/src/main/asciidoc/ch01.asciidoc

```diff
@@ -94,9 +94,9 @@ Starting with Hibernate Validator 9.0, the patch to update WildFly Jakarta Valid
 You can learn more about the WildFly patching infrastructure in general link:https://github.com/wildfly/wildfly-feature-pack-template[here].
 
 [[validator-gettingstarted-java9]]
-==== Running on Java 11+
+==== Running on Java {jdkVersion}+
```

Comment:
Seems weird that the title mentions a Java version, but the content is about JPMS.

Should probably be "Running in the modulepath" or "Running as a Java module"?

Reply:
+1 done 👍🏻 😃 

BTW, while we are at it (updating the docs)... should we also update `Jakarta Bean Validation` to `Jakarta Validation` to match what was done in the spec?

Reply:
> BTW, while we are at it (updating the docs)... should we also update `Jakarta Bean Validation` to `Jakarta Validation` to match what was done in the spec?

I think we should :/

Reply:
let me know if you'd want to take another look at this one 🙈 😏

Reply:
I didn't expect the "Jakarta Bean Validation" => "Jakarta Validation" change to happen in this PR, but okay :) Maybe a different issue though?

Also... There seem to be inconsistencies in the version of Jakarta Validation, as well as in the Java version (why talk about Java 11, don't we require 17?).